master/src/ComputerMonitoringWidget.cpp

```diff
@@ -271,6 +271,80 @@ void ComputerMonitoringWidget::runDoubleClickFeature( const QModelIndex& index )
 }
 
 
+void ComputerMonitoringWidget::runMousePressAndHoldFeature( )
+{
+   Q_EMIT mousePressAndHold( );
+   const auto selectedInterfaces = selectedComputerControlInterfaces();
+   if ( !m_ignoreMousePressAndHoldEvent &&
+		selectedInterfaces.count() > 0 &&
+		selectedInterfaces.count() < 2 &&
+		selectedInterfaces.first()->state() == ComputerControlInterface::State::Connected &&
+		selectedInterfaces.first()->hasValidFramebuffer() )
+   {
+		m_ignoreMousePressAndHoldEvent = true;
+		m_computerZoomWidget = new ComputerZoomWidget( selectedInterfaces.first()  );
+		QApplication::setOverrideCursor(Qt::BlankCursor);
+   }
+}
+
+void ComputerMonitoringWidget::stopMousePressAndHoldFeature( )
+{
+	m_computerZoomWidget->close();
+	QApplication::restoreOverrideCursor();
+}
+
+void ComputerMonitoringWidget::mousePressEvent( QMouseEvent* event )
+{
+	if(event->type() == QEvent::MouseButtonPress)
```

Comment:
I think you can skip this check. `mousePressEvent()` is only called for `QMouseEvents`.

Reply:
Okay. Done.